### PR TITLE
Fixing dependencies: removng janelia:render-app and correct imglib2 version

### PR DESCRIPTION
**Pull request by constantinpape**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -61,7 +61,7 @@
 		<dependency>
 			<groupId>net.imglib2</groupId>
 			<artifactId>imglib2</artifactId>
-			<version>3.1.0</version>
+			<version>3.2.1</version>
```
Comment (axtimwalde) [inline pom.xml:64]: We can use the managed version from parent pom 3.2.0

File: pom.xml [hunk 2]
```diff
@@ -119,11 +119,6 @@
 			<groupId>sc.fiji</groupId>
 			<artifactId>trakem2_tps</artifactId>
 		</dependency>
-		<dependency>
```
Comment (axtimwalde) [inline pom.xml:122]: Correct

**Comment by axtimwalde** [comment]
Instead of removing the render-app dependency, I commented it in master, the bump to imglib2 is covered by switching to parent managed versions for imglib2 and other dependencies that were previously overriding older version numbers

**Comment by constantinpape** [comment]
Ok, I have pulled your recent changes and this has fixed the problems.

**Pull request closed without merging**